libs/langchain/langchain/smith/evaluation/runner_utils.py

```diff
@@ -137,19 +137,6 @@ def _wrap_in_chain_factory(
                 "(memory=new_memory, ...)\n\n"
                 f'run_on_dataset("{dataset_name}", chain_constructor, ...)'
             )
-        logger.warning(
```

Comment:
This wasn't helpful i think. If no memory is specified it's not a concern, and we want more runnable use anyway